sailpoint-identitynow/operations.py

```diff
@@ -0,0 +1,171 @@
+""" Copyright start
+  Copyright (C) 2008 - 2022 Fortinet Inc.
+  All rights reserved.
+  FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
+  Copyright end """
+import json, base64
+from Crypto.PublicKey import RSA
+from Crypto.Cipher import PKCS1_v1_5 as Cipher_PKCS1_v1_5
+from connectors.core.connector import get_logger, ConnectorError
+from .oauth_token_handler import SailPointOAuth
+
+logger = get_logger('sailpoint-identitynow')
+
+
+class SailPointIdentityNow(object):
+    def __init__(self, config):
+        self.server_url = config.get('server_url', '').strip('/')
```

Comment:
Remove variables which are not used. 
if not self.server_url.startswith('https://') and not self.server_url.startswith('http://'):
            self._server_url = 'https://' + self.server_url
self.username = config.get('username')
        self.password = config.get('password')
        self.verify_ssl = config.get('verify_ssl')